src/test/scala-2.12/org/apache/james/gatling/jmap/scenari/JmapCountMailboxesScenario.scala

```diff
@@ -1,23 +1,22 @@
 package org.apache.james.gatling.jmap.scenari
 
-import io.gatling.core.Predef.{Simulation, atOnceUsers, nothingFor, rampUsers, scenario}
+import io.gatling.core.Predef.{Simulation, atOnceUsers, scenario, _}
```

Comment:
why?

Reply:
we are using `import io.gatling.core.Predef._` everywhere but here. It's seems legit as it is a DSL. Is it ok for you if I replace this by `import io.gatling.core.Predef._`?

Reply:
fixed everywhere